house_of_refuge/frontend/src/components/ResourceList.js

```diff
@@ -198,12 +200,14 @@ export const ResourceList = ({initialResources, sub, subHandler, user, clearActi
                 }}
             />
           </QuickFilter>
-          <QuickFilter>
-            <Search/>
-            <input className="search-input" onChange={(e) => setSearchQuery(e.target.value.toLowerCase())}/>
+          <QuickFilter label={"Szukaj"}>
+            <div className={"d-flex align-items-center"}>
+              <Search/>
+              <input className="search-input" onChange={(e) => setSearchQuery(e.target.value.toLowerCase())}/>
+            </div>
           </QuickFilter>
         </div>
-        <div className={"d-flex justify-content-around"}>
+        <div className={"d-flex justify-content-between align-items-center mx-5"}>
```

Comment:
this one i'm not sure about.. kinda liked it wasn't jsutified to the left.. It seems less noticeable here, wdyt about brining it closer to the pagination? Maybe it should actually be right beside it?

Reply:
Moved it to flex end 
<img width="376" alt="image" src="https://user-images.githubusercontent.com/6080725/159119262-3fcc00f1-7834-4a1c-9f08-ad9b5b5ea9d2.png">